src/qibolab/instruments/qblox/cluster_qrm_rf.py

```diff
@@ -18,7 +18,7 @@
 from .sweeper import QbloxSweeper, QbloxSweeperType
 
 
-class ClusterQRM_RF(ClusterModule):
+class QrmRf(ClusterModule):
```

Comment:
Thanks @PiergiorgioButtarini. I partly agree with the renames, particularly the removal of `_`. Maybe `QrmRF` would look better but no strong opinion from my side. Maybe we could also ask @aorgazf.

Other than that, could you please update the platforms in the other repo so that we can test?

Reply:
Yes I also prefer `QrmRF` but I wanted to follow @alecandido suggestions. I think also Alvaro will prefer this notation.
Sure I will open a PR with the updated runcards

Reply:
As I said, there is some arbitrariness involved on which acronym you want to turn into words. You could consider a single acronym as a whole, and just use `Qrmrf` (to me, it's the ugliest).

Within my personal arbitrariness, I'd pick the current one. But the one suggested by you is also fine.